### PR TITLE
Add internally hosted blog post

### PR DESCRIPTION
**Pull request by bryan-aguilar**
This PR adds a blog post announcement for ADOT release v0.16.0. Also updated is an outdated reference to the repository `master` branch. 

Read more redirects
image
To this static page

image

### CODE REVIEW

File: src/content/BlogPosts/blogPosts.yaml [hunk 1]
```diff
@@ -6,6 +6,14 @@ description:
 path: /blog
 
 blogs:
+  - title: "AWS Distro for OpenTelemetry 0.16 is now available with updated Lambda layers"
+    author: "Alolita Sharma and Raman Aulakh"
+    date: "31-Jan-2022"
+    body:
+      "AWS Distro for OpenTelemetry 0.16 is now available. You can download the latest AWS Distro for OpenTelemetry Collector image
+      from the Amazon Elastic Container Registry (Amazon ECR) Public Gallery."
+    link: "/docs/v0.16.0"
```
Comment (vasireddy99) [inline src/content/BlogPosts/blogPosts.yaml:15]: thanks for the PR, can we cross check the link, since the path for `v0.16.0` is under src/content/blogs/

Comment (bryan-aguilar) [reply src/content/BlogPosts/blogPosts.yaml:15]: The link is valid, I believe this is due to the way gatbsy is configured to auto generate .mdx files. Even though it exists in `content/blogs` MDX files are assigned a page at `/docs/*`. 

image

**Review by vasireddy99 — APPROVED** [review]

**Review by alolita — APPROVED** [review]
+1

Are there links for the highlights to docs that we could add?

File: gatsby-config.js [hunk 2]
```diff
@@ -51,6 +51,12 @@ module.exports = {
         path: `./src/content/BlogPosts`,
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `./src/content/blogs`,
```
Comment (jefchien) [inline gatsby-config.js:57]: Should probably be `./src/content/Blogs` for consistency. It's a bit confusing to have a `content/BlogPosts` and `content/Blogs`. On first glance, they seem like they would reference the same thing.

File: src/content/blogs/v0.16.0.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Arm 64 container images now available on [DockerHub](https://hub.docker.com/r/amazon/aws-otel-collector/tags) and [Amazon ECR](https://gallery.ecr.aws/aws-observability/aws-otel-collector)
+* Lambda layers for AWS X-Ray trace support for Java, Java auto-instrumentation, JavaScript, .Net, Go and Ruby; metrics support for managed service for Prometheus (AMP)
+* ADOT Collector performance testing improvements
+
+Detailed release notes are on [GitHub](https://github.com/aws-observability/aws-otel-collector/releases). All code changes are upstream in the respective OpenTelemetry project components.
+
+**Download**
+
+Detailed technical documentation is available on the [ADOT developer site](https://aws-otel.github.io/), and you can [download the distribution](https://aws-otel.github.io/download) from [GitHub](https://github.com/aws-observability/aws-otel-collector/releases/tag/v0.14.0). You can also download the latest [ADOT Collector image](https://gallery.ecr.aws/aws-observability/aws-otel-collector) from the [Amazon Elastic Container Registry (Amazon ECR)](https://aws.amazon.com/ecr/) Public Gallery.
```
Comment (jefchien) [inline src/content/blogs/v0.16.0.mdx:29]: The tag in this link should probably be v0.16.0.

**Review by alolita — APPROVED** [review]
+1

**Pull request merged**